ert3_examples/spe1/README.md

```diff
@@ -101,18 +109,27 @@ realisations. An important difference is that the ensemble size is no longer
 configured by the user, but dictated by the algorithm. There are also some
 differences in the `experiment.yml`, but those are hopefully self-explaining.
 
+#### Clean already ran experiments
+An experiment can only be run once. If you wish re-run your experiment, the
+results must be deleted first. To do this, execute `ert3 clean <experiment_name>`.
+To see the status of all the experiments in the workspace, run `ert3 status`.
+
+#### Visualize experiments
+After experiments have been evaluated. The records labled as output in the
+ensemble-config can be visualized through the visualization solution, `webviz-ert`.
+To launch the visualization tool run `ert vis`.
+
 ### Some additional notes
 ERT3 is highly experimental software in the sense that we aim at moving fast.
 Natural features that is lacking is among others:
  - inspect records and experiments in the cli,
- - delete records and experiment runs,
+ - delete workspace records,
  - monitor ongoing experiments,
- - visualize experiment results,
```

Comment:
Is this lacking?

Reply:
nope, not after #1743 is merged, that is why I am removing it.